.gitignore

```diff
@@ -0,0 +1 @@
+build
```

Comment:
Why not, but this is specific to your configuration, so I would not add this entry (e.g. I a generally using ``_build``, or ``_fastbuild`` etc, should we list all of them here?)